### PR TITLE
Akshaya p

### PR DESCRIPTION
**Pull request by AKSHAYAPALCHAMY-P**

### CODE REVIEW

File: AkshayaP/01 - HTML/01 - Basics/01 - Headings & Paragraphs/Headings.html
Comment (msindia-learning) [inline AkshayaP/01 - HTML/01 - Basics/01 - Headings & Paragraphs/Headings.html:1]: Use heading tags - h1 - h6

File: AkshayaP/01 - HTML/01 - Basics/03 - Links & Images/Links.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<body>
+    <!--absolute URL-->
+    <p> <a href="https://www.w3schools.com/html/" title="go to W3 schools HTML tutorial" target="_blank">Visit HTML tutorial</a></p>
+    <p><a href="https://www.google.com/">google</a></p>
+    <!--Relative URL-->
+    <p><a href="Paintings.htmlS7.jpg">PAINTINGS</a></p>
+    <p><a href="Paintings.htmlS7.jpg">PAINTINGS</a></p>
+    <p><a href="/html/default.asp">HTML tutorial</a></p>
+    <img src="Paintings.jpg" alt="HTML">
+    <button onclick="document.location='default.asp'">HTML tutorial</button>
```
Comment (msindia-learning) [inline AkshayaP/01 - HTML/01 - Basics/03 - Links & Images/Links.html:16]: <button onclick="document.location='default.asp'">HTML tutorial</button> 

Is this works?

File: AkshayaP/01 - HTML/01 - Basics/01 - Headings & Paragraphs/Headings.html
Comment (AKSHAYAPALCHAMY-P) [reply AkshayaP/01 - HTML/01 - Basics/01 - Headings & Paragraphs/Headings.html:1]: Done

File: AkshayaP/01 - HTML/01 - Basics/03 - Links & Images/Links.html (hunk 1 above)
Comment (AKSHAYAPALCHAMY-P) [reply AkshayaP/01 - HTML/01 - Basics/03 - Links & Images/Links.html:16]: Done

**Review by msindia-learning — APPROVED** [review]

**Pull request merged**